src/components/actions/index.scss

```diff
@@ -58,3 +60,15 @@
   box-shadow: none;
   text-decoration: none;
```

Comment:
I think it would look nice for the action text to never wrap. Since almost all of them are so small, having just one that is long and wraps looks weird to me. Let's try it?

Reply:
I'll try it as well

Reply:
Done!